backend/voting/models/ranking.py

```diff
@@ -35,9 +36,15 @@ def __str__(self):
         return f"{self.type} ({self.name}) at <{self.rank_request.conference.code}> | {self.value}"
 
 
+class Rank(TypedDict):
+    submission_id: int
+    submission__tag: SubmissionTag
+    score: float
+
+
 class RankRequest(models.Model):
 
-    conference = models.OneToOneField(
+    conference = models.ForeignKey(
```

Comment:
I relaxed this constraint to be able to see both the old ranking and the new one with the tags in order to check them, but I think I will revert this, we don't need to have multiple rankings for one conference.